packages/browser/src/Component/ItownsUtil.js

```diff
@@ -71,23 +32,9 @@ export function setup3DTilesLayer(layer, layerManager, itownsView) {
         url: layer['url'],
       }),
       registeredExtensions: extensions,
-      overrideMaterials: overrideMaterial,
     },
     itownsView
   );
-  if (overrideMaterial) {
-    $3dTilesLayer.overrideMaterials = material;
-    $3dTilesLayer.material = material;
-  }
-
-  const $3DTilesManager = new TilesManager(itownsView, $3dTilesLayer);
-
-  if (layer['color']) {
```

Comment:
Deleting those lines make it impossible to change the color of a tileset using the config file : is it a feature that we want to drop ? 
Same remark about point size of point cloud

Reply:
For the point cloud I will test the fresh new example and see if nothing is broken with the new implem. One thing I am thinking about is that overrideMaterials is now deprecated in itowns in order to force user to use style but I am not sure that style is compatible with point cloud and I need to see if it could if not I 'll just ask to itowns to restore the overrideMaterials attribute for the long term for that case (ie point cloud)

Reply:
Style in json file will come back with itowns then, fine for me.